coreutils/Makefile

```diff
@@ -470,7 +470,7 @@ csv-exp-201609:
 		done ; \
 	done
 
-# This target is used for running coreutils experiment with LLBMC.
+# This target is used for running coreutils experiment with LLBMC. #Arguments in PROGRAM_ARGS have lengths matching that of KLEE runs.
```

Comment:
I think the earlier comment was fine, I meant perhaps just to add another line for extra explanation.